image/image.html

```diff
@@ -3,12 +3,54 @@
     category: 'output',
     color: '#a6bbcf',
     defaults: {
+      active: {value:true},
       name: { value: "" }
     },
     inputs: 1,
     outputs: 0,
     icon: "watch.png",
     align: 'right',
+    button: {
+        toggle: "active",
+        onclick: function() {
+            var label = this.name||"debug";
+            var node = this;
+            $.ajax({
+                url: "image-output/"+this.id+"/"+(this.active?"enable":"disable"),
+                type: "POST",
+                success: function(resp, textStatus, xhr) {
+                    var historyEvent = {
+                        t:'edit',
+                        node:node,
+                        changes:{
+                            active: !node.active
+                        },
+                        dirty:node.dirty,
+                        changed:node.changed
+                    };
+                    node.changed = true;
+                    node.dirty = true;
+                    RED.nodes.dirty(true);
+                    RED.history.push(historyEvent);
+                    RED.view.redraw();
+                    if (xhr.status == 200) {
+                        RED.notify(node._("image-output.notification.activated",{label:label}),"success");
+                    } else if (xhr.status == 201) {
+                        RED.notify(node._("image-output.notification.deactivated",{label:label}),"success");
+                    }
+                },
+                error: function(jqXHR,textStatus,errorThrown) {
+                    if (jqXHR.status == 404) {
+                        RED.notify(node._("common.notification.error", {message: node._("common.notification.errors.not-deployed")}),"error");
```

Comment:
Would it make sense to abstract the `RED.notify(node._("common.notification.error", {message: ` part to a function to reduce repetition?

Reply:
Yep, I have changed it.  I have also replaced the locale paths (like e.g.  *common.notification.error*) by a fixed text.  Because the locales might change in the future, resulting in corrupting our code.   Have used your template literals all over the place ;-)

Here are some test results:

![image](https://user-images.githubusercontent.com/14224149/36816180-558fda52-1cdd-11e8-8f21-f920840b9780.png)